docs/index.asciidoc

```diff
@@ -424,6 +425,17 @@ When the `target` is set to a field reference, the `_source` of the hit is place
 This option can be useful to avoid populating unknown fields when a downstream schema such as ECS is enforced.
 It is also possible to target an entry in the event's metadata, which will be available during event processing but not exported to your outputs (e.g., `target \=> "[@metadata][_source]"`).
 
+
+[id="plugins-{type}s-{plugin}-retries"]
+===== `retries`
+
+* Value type is <<number,number>>
+* Default value is `1`
```

Comment:
Is it correct that by default we force a retry or should it be 0 by default?

Reply:
good catch Andrea, it should be 0, no retry